sw/device/silicon_creator/lib/sigverify/sphincsplus/hash.h

```diff
@@ -25,11 +25,10 @@ extern "C" {
  *
  * Appears in reference code as `initialize_hash_function`.
  *
- * @param ctx Context object.
  * @return Error code indicating if the operation succeeded.
  */
 OT_WARN_UNUSED_RESULT
-rom_error_t spx_hash_initialize(spx_ctx_t *ctx);
```

Comment:
I suspect the original author of this code was going for uniformity of interface (with respect to the context object).  Since @jadephilipoom is currently reworking the spx code, I'd like her input on eliminating the context parameter here and below.

Reply:
This should not be eliminated; it helps maintain uniformity with the reference implementation, and for some parameter sets it's useful; the open PR https://github.com/lowRISC/opentitan/pull/23761 will start using it.